docs/cloud/faq/providers.md

```diff
@@ -0,0 +1,14 @@
+# Providers
+
+### Azure: Do you use Locally redundant storage (LRS) or Zone-redundant storage (ZRS) ?
+
+Customer data is only stored on Premium SSD block device targets. 
```

Comment:
I'm not an Azure expert, so I'm not sure if the answer is yes or no 😅

Reply:
https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy
LRS & ZRS is Blob storage redundancy, if i understand correctly the data might be replicated in antohet region, what some customer do no want explicitely .

 so we use attached disk to the VMs  that are in the same regions/ datacenter and do not move to another region 
 https://docs.microsoft.com/en-us/azure/virtual-machines/managed-disks-overview